cli/events-plugin.md

```diff
@@ -0,0 +1,328 @@
+
+    ![](../img/cli-org-list.png)
+
+    To select an organization, use ***aio console org select <ims_org_id>***.
+
+    ![](../img/cli-org-select.png)
+
+
+2. Select the project you created in Console. You can list the projects using ***aio console project list***.
+
+    ![](../img/cli-project-list.png)
+
+    To select a project, use ***aio console project select <project_id>***.
+
+    ![](../img/cli-project-select.png)
+
+3. Select the workspace from the project you created in Console. By default a Production workspace is created when you create a new project. You can list the workspaces using ***aio console workspace list***.
+
+    ![](../img/cli-workspace-list.png)
+
+    To select a workspace, use ***aio console workspace select <workspace_id>***.
+
+    ![](../img/cli-workspace-select.png)
+
+## Installing and using the plugin
+
+Use the following syntax to install the Events Plugin for the Adobe I/O CLI and issue commands.
+
+```shell
+$ npm install -g @adobe/aio-cli-plugin-events
+$ aio COMMAND
+running command...
+$ aio (-v|--version|version)
+@adobe/aio-cli-plugin-events/1.0.0 darwin-x64 node-v10.18.1
+$ aio [COMMAND] --help
+USAGE
+  $ aio COMMAND
+...
+```
+
+## Commands
+
+There are a number of commands available when using the Events Plugin, please select an area from the following list to learn more: 
+
+* [Common command](#common-command)
+* [Registration commands](#registration-commands)
+* [Provider commands](#provider-commands)
+* [Event Metadata commands](#event-metadata-commands)
+
+### Common command
+
+```shell
+aio event
+```
+
+### Registration commands
+
+* [Manage your Adobe I/O Events Registrations](#manage-your-adobe-io-events-registrations)
+* [Get an Event Registration in your workspace](#get-an-event-registration-in-your-workspace)
+* [List Event Registrations in your workspace](#list-event-registrations-in-your-workspace)
+* [Create an Event Registration](#create-an-event-registration)
+
+#### Manage your Adobe I/O Events Registrations
+
+```shell
+USAGE
+$ aio event:registration
+ 
+OPTIONS
+-v, --verbose Verbose output
+--help Show help
+--version Show version
+```
+
+#### Get an Event Registration in your workspace
+
+```shell
+USAGE
+  $ ./bin/run event:registration:get REGISTRATIONID
+ 
+ARGUMENTS
+  REGISTRATIONID  Id of the registration to get
+ 
+OPTIONS
+  -j, --json     Output json
+  -v, --verbose  Verbose output
+  -y, --yml      Output yml
+  --help         Show help
+  --version      Show version
+ 
+ALIASES
+  $ ./bin/run console:reg:get
+```
+
+#### List Event Registrations in your workspace
+
+```shell
+USAGE
+$ ./bin/run event:registration:list
+ 
+OPTIONS
+-j, --json Output json
+-v, --verbose Verbose output
+-y, --yml Output yml
+--help Show help
+--version Show version
+ 
+ALIASES
+$ ./bin/run console:registration:ls
+$ ./bin/run console:registration:list
```

Comment:
This has been corrected recently to event:reg:list. Also there was  mistake in the wiki. we should have event instead of console in all these aliases for registration. 

So the aliases in this case are :
ALIASES
  $ aio event:registration:ls
  $ aio event:reg:list
  $ aio event:reg:ls